### ISSUE TITLE
Adding Android OS 

### ISSUE THREAD

**Issue by ppeyso** [m3y54m/Embedded-Engineering-Roadmap#28]
Embedded Android is a worldwide technology that is used in many devices like smartphones, tablets, and smart TVs. So learning Android projects for well-known SOCs like Allwinner, rock-chips, etc would be a path in front of an embedded system engineer. I want to add related skills in Android development to your great roadmap. thanks
Best regards.
Peyso

**Comment by m3y54m** [comment]
That’s a great suggestion! However, I have limited experience with Embedded Android. If you know of any good learning resources, please feel free to share them here. Alternatively, you’re welcome to submit a pull request if you’d like to contribute directly.

**Comment by pavly-gerges** [comment]
Android Embedded Engineering is typically a variant of Embedded Linux Engineering in addition to some Android Application development skills (e.g., Android SDK and NDK). I could initiate this section and just readings for it as necessary. However, you should know that knowing Embedded Linux beforehand would be the rule.

**Comment by pavly-gerges** [comment]
> Just a quick note, there’s already an **"Embedded Linux"** section on the roadmap, as well as in the curated resources list. It would be perfect to add "Embedded Android" as a sub-section under "Embedded Linux".

I agree. Furthermore, it would be nice of you if you add the Microsoft Visio File to this repository that is responsible for the digital roadmap, so new people building PRs would be able to update the map as necessary. Thanks for making this public!
